resource-managers/kubernetes/core/src/main/scala/org/apache/spark/deploy/kubernetes/submit/submitsteps/PythonStep.scala

```diff
@@ -30,7 +30,7 @@ private[spark] class PythonStep(
     val resolvedOtherPyFilesString = if (otherPyFiles.isEmpty) {
       "null"
     } else {
-      otherPyFiles.mkString(",")
+      KubernetesFileUtils.resolveSubmittedUris(otherPyFiles, filesDownloadPath).mkString(",")
```

Comment:
I understand what you are saying. Although, when I did testing before I didn't see the remote files downloaded into `/var/spark-data/`. Is this handled with the newest push? If it is, then, yeah it should used resolveFilePaths()